pom.xml

```diff
@@ -0,0 +1,405 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.strimzi</groupId>
+    <artifactId>mirror-maker-2-extensions</artifactId>
+    <version>0.20.0-SNAPSHOT</version>
```

Comment:
Maybe the version should be 0.1.0-SNAPSHOT? Or 1.0.0-SNAPSHOT? I think it makes no sense to version it based on the operators versions since we will release this only as needed.